README.md

```diff
@@ -860,6 +860,8 @@ registration: {
 
 ## Registration
 
+> **:warning: Beta warning:** The registration feature is currently in beta. This widget functionality won't work unless your Okta organization is part of the registration beta program. For help, contact support@okta.com.
+
 To add registration into your application, configure your Okta admin settings to allow users to self register into your app. Then, set `features.registration` in the widget. You can add additional configs under the registration key on the [`OktaSignIn`](#new-oktasigninconfig) object.
```

Comment:
Can we link to the doc that explains the Beta program and capitalize Beta? https://developer.okta.com/docs/api/getting_started/releases-at-okta#beta has the link to the signup page.
I'm assuming our liquid tags for Beta won't work here {% api_lifecycle beta %}

Reply:
Good suggestion on the link, thanks!